guides/v2.3/release-notes/2.3.0-alpha-install.md

```diff
@@ -9,37 +9,41 @@ github_link: release-notes/2.3.0-alpha-install.md
 
 In addition to the [standard Magento prerequisites]({{ site.baseurl }}/guides/v2.3/install-gde/prereq/prereq-overview.html), make sure you have obtained and installed:
 
-* *Authentication keys*. The {{site.data.var.ce}} 2.3.0 Alpha Composer package is available to everyone. For access to the {{site.data.var.ee}} 2.3.0 Alpha Composer package, contact [Magento Support](https://magento.com/support). See [Get your authentication keys]({{page.baseurl}}/install-gde/prereq/connect-auth.html) for more information.
+* *Open Source Access*. The Open Source 2.3.0 Alpha Composer package is available to everyone with Magento authentication keys. If you do not have authentication keys, see [Get your authentication keys]({{page.baseurl}}/install-gde/prereq/connect-auth.html) for more information.
+* *Commerce Access*. The Commerce 2.3.0 Alpha Composer package is only available to Magento partners, as well as approved merchant developers and independent extension developers. In addition, access is granted only to those who complete and sign [Magento's Pre-Release Software Agreement](https://partners.magento.com/portal/pre-release-agreement). You will be notified of your approval to access the {{site.data.var.ee}} 2.3.0 Alpha repository within three business days. After you receive approval you will be able to install {{site.data.var.ee}} 2.3.0 Alpha using these installation instructions.
 * *PHP 7.1 or 7.2 and extensions*. See instructions here: [Install PHP 7.1 for Ubuntu]({{ site.baseurl }}/guides/v2.3/install-gde/prereq/php-ubuntu.html#instgde-prereq-php71-ubuntu) or [Install PHP 7.1 for Centos]({{ site.baseurl }}/guides/v2.3/install-gde/prereq/php-centos.html#php-centos-71).
 * *Composer*. Install using:
     ```bash
     curl -sS https://getcomposer.org/installer | sudo php -- --install-dir=/usr/bin --filename=composer
     ```
 
 ## Install 2.3.0 Alpha
-  
-{:.bs-callout .bs-callout-info}
-Currently, the Magento 2.3.0 Alpha is only available for Open Source modules. Alpha for Commerce, including the Magento Commerce for B2B extension, will be made available soon.
 
-1. Install the Open Source or Commerce alpha package.
+1. Install the Open Source or Commerce Alpha package.
 
    <div class="bs-callout bs-callout-warning" id="info" markdown="1">
```

Comment:
We have moved away from the **2.3.0-alpha** label and are now using numbered alphas (e.g. **2.3.0-alpha55**), so this warning is no longer necessary.

Reply:
Great. I'll remove.